### PR TITLE
Removed unused files

### PR DESCRIPTION
**Pull request by NeoChen1024**

### CODE REVIEW

File: vendor/LICENSE [hunk 1]
```diff
@@ -1,21 +0,0 @@
-The MIT License (MIT)
```
Comment (lwhsu) [inline vendor/LICENSE]: Don't remove other's license.

**Review by lwhsu — CHANGES_REQUESTED** [review]

File: vendor/LICENSE (hunk 1 above)
Comment (NeoChen1024) [reply vendor/LICENSE]: That license is same as ours (verified with diff(1))

**Comment by NeoChen1024** [comment]
LICENSE added back

File: vendor/LICENSE (hunk 1 above)
Comment (lwhsu) [reply vendor/LICENSE]: Oh I was not aware we have another LICENSE file in root, then it's OK to remove this one.

**Comment by lwhsu** [comment]
Do you want to git rebase and squash the commits?

Ref: https://backlog.com/git-tutorial/tw/stepup/stepup7_5.html

**Comment by NeoChen1024** [comment]
Squash completed

**Pull request closed without merging**